docusaurus/docs/dev-docs/plugins/upload.md

```diff
@@ -172,6 +172,50 @@ export default {
 
 </Tabs>
 
+### Upload request timeout
+
+By default, the value of `strapi.server.httpServer.requestTimeout` is set to 330 seconds. This includes uploads. To make it possible for users with slow internet connection to upload large files, it might be required to increase this timeout limit by setting it in the bootstrap function that runs before strapi gets started.
```

Comment:
```suggestion
By default, the value of `strapi.server.httpServer.requestTimeout` is set to 330 seconds. This includes uploads. To make it possible for users with slow internet connection to upload large files, it might be required to increase this timeout limit. The recommended way to do it is by setting the `http.serverOptions.requestTimeout` parameter in the `config/server.js|ts` file (see [server configuration](/dev-docs/configurations/server).
An alternate method is to set the `requestTimeout` value in the `bootstrap` function that runs before Strapi gets started. This is useful in cases where it needs to change programmatically — for example, to temporarily disable and re-enable it:
```

Reply:
I've enhanced your content to reflect the comment from @innerdvations on PR #1903 :)